plugin/yanks.vim

```diff
@@ -0,0 +1,16 @@
+" vim-clap - Modern interactive filter and dispatcher
```

Comment:
Is it better to put this files in plug/clap.vim?

```vim
if get(g:, 'clap_enable_yanks_provider', 1)
  augroup ClapYanks
    autocmd!
    autocmd VimEnter * call clap#provider#yanks#init()
  augroup END
endif
```

Reply:
Done!